pkg/daemon/struct.go

```diff
@@ -200,6 +202,21 @@ func (p *Pod) GetServiceMeta(n string) map[string]string {
 	return nil
 }
 
+// GetServiceHealth returns the service health specified in annotation, or defaultVal if not specified.
+func (p *Pod) GetServiceHealth(n string, defaultVal string) string {
+	healthStr := p.Pod.ObjectMeta.Annotations[ConsulServiceHealthOverride+n]
+	if healthStr == "" {
+		healthStr = p.Pod.ObjectMeta.Annotations[ConsulServiceHealth]
+	}
+	switch healthStr {
```

Comment:
nit: We may want to have a separate `default` switch case here if `n` is != "" to log an error; otherwise this might be a bit difficult to debug

Reply:
done